<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>reg_iva</class>
 <widget class="QDialog" name="reg_iva">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>509</width>
    <height>409</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>509</width>
    <height>409</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>509</width>
    <height>409</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Stampa registro iva</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../fabaria.qrc">
    <normaloff>:/images/iva.png</normaloff>:/images/iva.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout_7">
   <item row="3" column="2">
    <widget class="QPushButton" name="esci">
     <property name="maximumSize">
      <size>
       <width>396</width>
       <height>320</height>
      </size>
     </property>
     <property name="baseSize">
      <size>
       <width>396</width>
       <height>320</height>
      </size>
     </property>
     <property name="text">
      <string>Esci</string>
     </property>
     <property name="icon">
      <iconset resource="../fabaria.qrc">
       <normaloff>:/images/dialog-close.png</normaloff>:/images/dialog-close.png</iconset>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QPushButton" name="report">
     <property name="text">
      <string>Report</string>
     </property>
     <property name="icon">
      <iconset resource="../fabaria.qrc">
       <normaloff>:/images/report.png</normaloff>:/images/report.png</iconset>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <spacer name="horizontalSpacer_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0" colspan="3">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Credito/Debito</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_6">
      <item row="0" column="0">
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Iva a Credito/Debito dal periodo precedente</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="iva_prec"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Data da usare per ordinare Fatt. Acquisto/vendita</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QComboBox" name="comboBox">
          <item>
           <property name="text">
            <string>Data registrazione</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Numero documento</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Periodo</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_5">
      <item row="0" column="0">
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="1">
         <widget class="QComboBox" name="tipo_fattura">
          <item>
           <property name="text">
            <string>Fattura di acquisto</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Fattura di ventita</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Seleziona tipo di fattura</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Seleziona il periodo</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QComboBox" name="mese"/>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Anno</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="anno"/>
        </item>
       </layout>
      </item>
      <item row="0" column="1">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0" colspan="3">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Liquidazione</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="1">
         <widget class="QRadioButton" name="mensile">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Mensile</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Tipo liquidazione</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QRadioButton" name="trim">
          <property name="text">
           <string>Trimestrale</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QRadioButton" name="annuale">
          <property name="text">
           <string>Annuale</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>mensile</tabstop>
  <tabstop>trim</tabstop>
  <tabstop>annuale</tabstop>
  <tabstop>tipo_fattura</tabstop>
  <tabstop>mese</tabstop>
  <tabstop>anno</tabstop>
  <tabstop>iva_prec</tabstop>
  <tabstop>comboBox</tabstop>
  <tabstop>report</tabstop>
  <tabstop>esci</tabstop>
 </tabstops>
 <resources>
  <include location="../fabaria.qrc"/>
 </resources>
 <connections/>
</ui>
